src/main/default/lwc/voteCount/voteCount.js

```diff
@@ -0,0 +1,68 @@
+import { LightningElement, api } from 'lwc';
+import helpedUsers from '@salesforce/label/c.cookbook_HelpedUsers';
+import helpedUsers_plural from '@salesforce/label/c.cookbook_HelpedUsers_plural';
+import helpedUsers_zero from '@salesforce/label/c.cookbook_HelpedUsers_zero';
+import { I18nUtils } from 'c/quanticUtils';
+
+/**
+ * The `voteCount` component is an indicator of how many people said that a document was helpful.
```

Comment:
```suggestion
 * The `voteCount` component is an indicator of how many people reported that a document was helpful.
```

It might be nice to also mention that the `voteButton` should be used to cast a vote.